modules/gapi/test/common/gapi_imgproc_tests_inl.hpp

```diff
@@ -50,6 +50,25 @@ namespace
             rgb2yuyv(in_line_p, out_line_p, in.cols);
         }
     }
+
+    void FormTrackingPointsArray(vector<Point2f>& points, int width, int height, int nPointsX, int nPointsY)
```

Comment:
Please name the function from the lower case letter.
Can this function somehow be shared between accuracy and performance tests?